site/content/docs/main/supported-providers.md

```diff
@@ -56,40 +61,7 @@ In the case you want to take volume snapshots but didn't find a plugin for your
 [4]: https://github.com/StackPointCloud/ark-plugin-digitalocean
 [5]: http://www.noobaa.com/
 [6]: https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/main/backupstoragelocation.md
-[7]: https://aws.amazon.com
```

Comment:
any specific reasons to remove these but preserve 0-6, 25, 36, 38?

Reply:
i took out the links in the tables, but left the references for the rest of thee links outside the table. We can convert all the links to be inline, but i was just focusing on fixing the tables in my first pass

Reply:
As a technical explanation, reference links (ones with the numbers) aren't passed through to the shortcodes files, so we have to have the URL directly in the link when using them in the table shortcode.

This is something that has to do with the contexts on the Markdown renderers that Hugo uses - they only have context in the file they're processing, shortcodes don't get the link numbering for the page they're being included in.